index.html

```diff
@@ -32,7 +32,8 @@
 	<script>
 		mapboxgl.accessToken = 'pk.eyJ1IjoiamFjb2JyZGFsdG9uIiwiYSI6ImNrYXc2anFjbjIxNGwyeG14dWk0MDVycmIifQ.yILWplracVMB4mv1dWwTtg'
 		const DATA_URL = 'https://spreadsheets.google.com/feeds/list/1CyPozeKhmOuIaVnKDQAUKKsvA9R4F20hEQ3MSGWczP8/1/public/full?alt=json'
-	</script>
+		const TRANS_URL = 'https://spreadsheets.google.com/feeds/list/1m5QPNP6O8nsQsqJcQbwib_obpXRjmbYYGtwKzzg1l38/1/public/full?alt=json'
```

Comment:
Is this basically a copy of the original spreadsheet that is translated whole hog? Are just parts translated?

Reply:
This is a nitpick but I think using `TRANSLATION_URL` for the variable name, while longer, would be more appropriate.

Reply:
@jdalt only static labels like column headers, map legend, etc are being translated. Translating the actual data is a huge lift but this gets us a step closer